h2o-core/src/main/java/water/rapids/ast/prims/mungers/AstSort.java

```diff
@@ -17,15 +19,22 @@
 public class AstSort extends AstPrimitive {
   @Override public String[] args() { return new String[]{"ary","cols"}; }
   @Override public String str(){ return "sort";}
-  @Override public int nargs() { return 1+2; } // (sort ary [cols])
+  @Override public int nargs() { return 1+2+1; } // (sort ary [cols] [boolean])
 
   @Override public ValFrame apply(Env env, Env.StackHelp stk, AstRoot asts[]) {
     Frame fr = stk.track(asts[1].exec(env)).getFrame();
     int[] cols = ((AstParameter)asts[2]).columns(fr.names());
+    int[] sortAsc;
+    if (asts[3] instanceof AstNumList)
+      sortAsc = ((AstNumList) asts[3]).getIntBases();
```

Comment:
Can you use the existing AstNumList.expand() method instead?

Reply:
The expand method returns a double array.  I want a integer array so that I can multiply it with long type and double types down the road without having to manually cast it.

Reply:
I think it would be better to use the existing expand() function and convert the output to integers - such a method can be implemented on AstNumList. Eg.: expandToInts

better than to duplicate the code

Reply:
Let me try it.

Reply:
I think you are talking about expand4().  It returns an array of integers.

Reply:
Great, please use that!